changelogs/unreleased/4803-compiler-service-notification.yml

```
@@ -0,0 +1,7 @@
+description: configure notification behavior for compile request
```

When we use this as a message in the actual changelog we should be a bit more verbose.
```suggestion
description: Added options to compiler service to configure notification behavior
```